tools/kvmexit.py

```diff
@@ -258,17 +193,53 @@ def valid_args_list(args):
     raise Exception("Failed to do precondition check, due to: %s." % e)
 
 try:
-    if BPF.support_raw_tracepoint_in_module():
-        # Let's firstly try raw_tracepoint_in_module
+    # Let's firstly try raw_tracepoint_module
+    #if support_raw_tracepoint_module():
+    try:
```

Comment:
Any special consideration about removing the `BPF.support_raw_tracepoint_in_module()` check?